hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/operation/HederaCreateOperationExternalizer.java

```diff
@@ -16,9 +16,7 @@
 
 package com.hedera.node.app.service.mono.contracts.operation;
 
-import static com.hedera.node.app.service.mono.ledger.properties.AccountProperty.ETHEREUM_NONCE;
-import static com.hedera.node.app.service.mono.ledger.properties.AccountProperty.IS_SMART_CONTRACT;
-import static com.hedera.node.app.service.mono.ledger.properties.AccountProperty.KEY;
+import static com.hedera.node.app.service.mono.ledger.properties.AccountProperty.*;
```

Comment:
We don't use `*` imports - but I think IntelliJ might have a setting where it automatically moves to `*` imports when you get more than a certain number (like 3) - and it's invisible usually because the imports are usually collapsed.  Please move _back_ to individual imports and adjust your IntelliJ settings!